src/index.js

```diff
@@ -48,13 +48,19 @@ async function start(fields) {
 async function waitFor2FACode() {
   const timeout = startTime + 3 * 60 * 1000
   let account = {}
+
+  // init code to null in the account
+  await this.updateAccountAttributes({
+    '2fa_code': null
```

Comment:
Let use `twofa_code` as name here to avoid using this name as variable like `['2fa_code']` since digit is not allowed as first name letter

Reply:
Same thing for 2FA_NEEDED, it will be better to not use digit as first letter

Reply:
https://github.com/konnectors/dummy/pull/146